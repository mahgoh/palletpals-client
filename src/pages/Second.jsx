import Main from '@/components/Main'
import Pagetitle from '@/components/Pagetitle'
import { useTranslation } from 'react-i18next'

export default function Second() {
  const { t } = useTranslation()
  return (
    <Main>
      <Pagetitle title={t('common.second')} />
    </Main>
  )
}
