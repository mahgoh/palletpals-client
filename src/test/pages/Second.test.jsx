import React from 'react'
import renderer from 'react-test-renderer'
import { toJson } from '@/test/util'
import Second from '@/pages/Second'

test('Render page Second', () => {
  const component = renderer.create(<Second />)

  let tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
