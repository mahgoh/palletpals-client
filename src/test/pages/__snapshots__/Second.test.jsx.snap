// Vitest Snapshot v1

exports[`Render page Second 1`] = `
<main
  className="mx-auto py-12 px-4 lg:max-w-6xl lg:px-10"
>
  <div>
    <h1
      className="text-4xl font-extrabold leading-10 tracking-tight sm:text-6xl sm:leading-none lg:text-5xl"
    >
      common.second
    </h1>
  </div>
</main>
`;
